src/test/java/seedu/address/logic/commands/TagListCommandTest.java

```diff
@@ -0,0 +1,29 @@
+package seedu.address.logic.commands;
+
+import static seedu.address.logic.commands.CommandTestUtil.assertCommandSuccess;
+import static seedu.address.testutil.TypicalPersons.getTypicalAddressBook;
+import static seedu.address.testutil.TypicalTags.FRIENDS;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.model.Model;
+import seedu.address.model.ModelManager;
+import seedu.address.model.UserPrefs;
+
+/**
+ * Contains tests for TagListCommand.
+ */
+public class TagListCommandTest {
```

Comment:
Can consider adding test cases for invalid inputs like `taglist test`, `taglist 123` etc.